ghost/admin/app/components/gh-explore-iframe.hbs

```diff
@@ -1 +1,9 @@
-<iframe id="explore-frame" class="explore-frame" frameborder="0" title="Explore" ...attributes {{did-insert this.setup}}></iframe>
\ No newline at end of file
+<iframe
+    id="explore-frame"
+    class="explore-frame"
+    frameborder="0"
+    title="Explore"
+    {{did-insert this.setup}}
+    {{did-insert this.attachMessageListener}}
```

Comment:
[`{{did-insert}}` is an anti-pattern](https://github.com/emberjs/ember-render-modifiers#when-to-use-these-modifiers-and-when-not-to-use-them) and we shouldn't introduce any more of them. In this case its use causes potential problems because you're attaching an event listener to `window` but there's no cleanup so if the iframe gets rendered and destroyed multiple times you'll have multiple event handlers responding to the iframe message event.

Two typical routes to go when you're reaching for `{{did-insert}}`:
1. if you're using the `{{did-insert}}` modifier to get access to the element then consider adding a custom modifier, especially if the behaviour is fairly generic (e.g. `{{on-window-event "message" this.handleIframeMessage}}`)
2. if you're not using the element and you're working in a component then you can use the component's lifecycle with its `constructor()` and `willDestroy()` hooks (probably the simpler option in this case)